niceman/distributions/conda.py

```diff
@@ -27,6 +27,37 @@
 import logging
 lgr = logging.getLogger('niceman.distributions.conda')
 
+# Provides the conda platform mapping from comda/models/enum.py
+# Note that these are python prefixes (both 'linux2' and 'linux' from python
+# map to 'linux' in conda.
+python_to_conda_platform_map = {
+    'darwin': 'osx',
+    'linux': 'linux',
+    'openbsd': 'openbsd',
+    'win': 'win',
+    'zos': 'zos',
+}
+
+
+def get_conda_platform_from_python(py_platform):
+    """
+    Converts a python platform string to a corresponding conda platform
+
+    Parameters
+    ----------
+    py_platform : basestring
```

Comment:
s/basestring/str/, since basestring doesn't exist in Python 3.

Or "string", I guess.  Our docstrings are inconsistent:

```
$ git grep " : str\b" | wc -l
14
$ git grep " : string" | wc -l
40
```

"string" is the clear winner (especially since a few of those "str"s are mine), but I prefer "str" because it matches the actual type.  "str" is also consistent with our preference for "bool" over "boolean" and "dict" over "dictionary".

```
$ git grep " : bool\b" | wc -l
15
$ git grep " : boolean" | wc -l
2
$ git grep " : dict\b"  | wc -l
15
$ git grep " : dictionary"  | wc -l
2
```

Another point in favor of "str": The [numpy documentation][0] also uses it.

[0]: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt

Reply:
pycharm suggested basestring, but I prefer str myself.